hasher-matcher-actioner/hmalib/common/config.py

```diff
@@ -366,3 +404,23 @@ def _config_to_dynamodb_item(config) -> t.Dict[str, t.Any]:
     item["ConfigType"] = config.get_config_type()
     item["ConfigName"] = config.name
     return item
+
+
+def _get_update_params(body: dict):
```

Comment:
How does this handle types?

Given that we are accepting dict in the update_config_attributes_by_type_and_name, it is possible that type information could be lost. 

In [Amazon's example](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.Python.03.html#GettingStarted.Python.03.03), they use the Decimal type when adding a number. We'd need to do something similar, right?

Reply:
Also, this method's returns are not typed. In any new code we are writing, we must have types.